src/constants/spacing.js

```diff
@@ -0,0 +1,5 @@
+export const letterSpacing = {
```

Comment:
I think we can keep letterSpacing inside `AbstractText.index.js` its not used outside that component.